extracts_v6.go

```diff
@@ -1167,6 +1169,17 @@ func extractFleetsFromDocV6(doc *goquery.Document, clock clockwork.Clock) (res [
 			secs = 0
 		}
 
+
+		var startTimeString string
+		if !returnFlight {
+			startTimeString, _ = s.Find("div.origin img").Attr("title")
+		} else {
+			startTimeString, _ = s.Find("div.destination img").Attr("title")
+		}
+		startTimeArray := strings.Split(startTimeString, ":| ")
+		startTime, _ := time.ParseInLocation("02.01.2006<br>15:04:05", startTimeArray[1], servertime.Location())
```

Comment:
Maybe for safety we need to check if startTimeArray[1] exists with len() == 2? :

```suggestion
	if !startTimeStringErr {
		startTimeArray := strings.Split(startTimeString, ":| ")
		if len(startTimeArray) == 2 {
			startTime, _ := time.ParseInLocation("02.01.2006<br>15:04:05", startTimeArray[1], servertime.Location())
		}
	}
```